src/vs/code/electron-main/window.ts

```diff
@@ -319,6 +321,48 @@ export class CodeWindow extends Disposable implements ICodeWindow {
 		});
 	}
 
+
+	private registerSwipeNavigationListener(command: any, back: 'left' | 'browser-backward', forward: 'right' | 'browser-forward', config: 'actions' | 'recent-tabs' | 'tabs' | 'grouped-tabs'): void {
```

Comment:
I couldn't use `'swipe' | 'app-command'` type union here, since the definitions from Electron didn't recognize them for some reason. Open for suggestions on how to mitigate that. Adding `string` didn't work either.